### DISCUSSION TITLE
Welcome to scml Discussions!

### DISCUSSION THREAD

**Discussion by yasserfarouk** [yasserfarouk/scml#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by YueNing** [comment]
:boom: Hi everyone in scml community,:boom: 

very happy to have such a place for discussion
I am [naodongbanana](https://github.com/YueNing), master student of Karlsruhe Institute of Technology, Germany. 

Now i am trying to realize an agent  in SCMLOneShot with some methods in the field multi agents deep reinforcement learning.

If anyone interests in it and has some ideas. could join us, we could develop a better agent and learn more knowledge about reinforcement learning and concurrent negotiations. 

Best wishes :smile: :muscle:
[naodongbanana](https://github.com/YueNing)
